fawltydeps/settings.py

```diff
@@ -429,15 +460,17 @@ def setup_cmdline_parser(
     ).add_mutually_exclusive_group()
     populate_output_formats(output_format_group)
 
+    # A group for source paths options
+    source_group = parser.add_argument_group(title="Source paths options")
+    populate_parser_paths_options(source_group)
+
+    # A group for fawltydeps configuration options
+    config_group = parser.add_argument_group(title="Configuration options")
+    populate_parser_configuration(config_group)
+
     # A different group for the other options.
     option_group = parser.add_argument_group(title="Other options")
     populate_parser_options(option_group)
-    option_group.add_argument(
-        "-h",
-        "--help",
-        action="help",
-        help="Show this help message and exit",
-    )
```

Comment:
The reason this was originally put in `settings` is that it is connected to this line, ~20 lines further up in `setup_cmdline_parser()`:
```
        add_help=False, # instead, add --help in the "Options" group below
```
That by itself is probably not a good enough reason to keep it here, but it's probably worth adding a comment about this coupling, something like:
```
    # setup_cmdline_parser() removes the automatic `--help` option so that we
    # can control exactly where it's added. Here we add it back:
```